test/lit.cfg

```diff
@@ -158,8 +158,16 @@ if platform.system() == 'Darwin':
     config.environment['TOOLCHAINS'] = \
         os.environ.get('TOOLCHAINS', config.darwin_xcrun_toolchain)
 
+
+kIsWindows = platform.system() == 'Windows'
```

Comment:
Nitpick: please match the naming conventions for all the other variables in the file (snake_case, no leading "k").

Reply:
This is a bit more complicated: this is also the name of a variable in the rest of lit (where I stole it from).  It is in the core lit infrastructure ... so I'm kinda torn (my personal preference is the snake case).

Reply:
Ah. Uh…we're *setting* a variable in the rest of lit? Is it expecting us to do that?

Basically, if it's what Clang does then okay (although it may be worth leaving a comment).